src/main/java/com/bootcamp/demo/data_access/Repo.java

```diff
@@ -0,0 +1,14 @@
+package com.bootcamp.demo.data_access;
+
+import com.bootcamp.demo.model.Account;
+
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+
+public interface Repo<T> {
```

Comment:
Ai putea redenumi Repo in Repository? :)

Reply:
Da, ma gandeam sa nu fie confusing pentru ca si spring are Repository